test/integration/simple_example/controls/forseti.rb

```diff
@@ -121,26 +108,17 @@
 
     its('object_names') { should include(*files) }
   end
-end
 
-control 'forseti-client-service-account' do
-  title 'Test Forseti Client Service Account'
   describe google_service_account(name: "projects/#{project_id}/serviceAccounts/#{forseti_client_service_account}") do
     its(:email) { should eq forseti_client_service_account }
     its(:display_name) { should eq "Forseti Client Service Account" }
   end
-end
 
-control 'forseti-server-service-account' do
-  title 'Test Forseti Server Service Account'
   describe google_service_account(name: "projects/#{project_id}/serviceAccounts/#{forseti_server_service_account}") do
     its(:email) { should eq forseti_server_service_account }
     its(:display_name) { should eq "Forseti Server Service Account" }
   end
-end
 
-control 'forseti-firewall-rules' do
```

Comment:
Why are we combining all controls into one?

Reply:
In order to target GCP and the hosts we need to assign specific controls to specific systems (https://github.com/terraform-google-modules/terraform-google-forseti/pull/48/files#diff-d3fbd21bc6be7ce4531c27a82b2619deR29); without this assignment we'll attempt to run the client and server controls with the GCP backend, which will fail. We could break out the controls as was done previously and add the list to `.kitchen.yml` - but we're not getting value from having fine grained controls. Pushing all of the example groups into a single control provides the functionality we need and doesn't have any real cost, aside from slightly different output formatting.

Reply:
Got it, thanks.